src/Http/Requests/CreatePromotion.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Vanilo\Admin\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Vanilo\Admin\Contracts\Requests\CreatePromotion as CreatePromotionContract;
+
+class CreatePromotion extends FormRequest implements CreatePromotionContract
+{
+    public function rules()
+    {
+        return [
+            'name' => 'required|string|min:1|max:255',
+            'description' => 'nullable|string|min:1|max:65000',
+            'priority' => 'required|integer|min:0',
```

Comment:
Could you please make the priority optional? And if there's no value passed, make it default to `10`

Reply:
I've addressed the request by using the `prepareForValidation` method to handle the priority field. If the user does not provide a value for priority, it defaults to `10`.